src/service.py

```diff
@@ -10,15 +10,16 @@
 import pandas as pd
 import urllib3
 
-
 class FastTextServiceBase(mlservicewrapper.core.services.Service):
 
     async def load(self, ctx: mlservicewrapper.core.contexts.ServiceContext):
         model_path = self.get_model_path(ctx)
-        
+
         if not os.path.exists(model_path):
+            model_parent_path = model_path.rsplit(os.sep, 1)[0]
```

Comment:
Better, but I can do you one better yet! Check out [`os.path.dirname`](https://docs.python.org/3/library/os.path.html#os.path.dirname). The functionality is the same, but it's purpose-built for doing this exact thing.

Reply:
Awesome!